lib/helper/Playwright.js

```diff
@@ -1468,11 +1517,36 @@ class Playwright extends Helper {
 
   /**
    * {{> clearField }}
+   * (deprecated) use `I.clear() instead`
    */
   async clearField(field) {
+    console.log('I.clearField() is DEPRECATED: use `I.clear()` to clear field or text area');
     return this.fillField(field, '');
   }
 
+  /**
+   * {{> clear }}
+   * Clear the <input>, <textarea> or [contenteditable] .
+   * @param {any} [options] [Additional options](https://playwright.dev/docs/api/class-locator#locator-clear) for available options object as 2nd argument.
+   *
+   * Examples:
+   *
+   * ```js
+   * I.clear('.text-area')
+   * ```
+   * ```js
+   * I.clear('#submit', { force: true }) // force to bypass the [actionability](https://playwright.dev/docs/actionability) checks.
+   * ```
+   *
+   */
+  async clear(locator, options = {}) {
```

Comment:
In a good way, we need to completely replace the `.clearField()`. 
`.clearField()` has already been used many times by users. 

#### Here are the possible options, as I see it:
1.  we need to either replace it slowly in the next Codeceptjs **major** version,
2. leave it as it is and remove deprecated in the next versions
3. completely replace the `.clearField()` here

So we need a final decision from @DavertMik

Reply:
I've the same opinion as you.
As these is a major change, need approval from @DavertMik

Reply:
> I'm up with replacing the implementation of clearField() instead of introducing different names. As far as I know the current clearField still brings some issues.

@EgorBodnar yes, I agree to replace implementation

It's not a major change as the behavior should be the same as current clearField
Sure, implementation is different and may bring some edge cases issue but I agree, no need to introduce new method

Reply:
replaced